lib/ListActions/ListActions.js

```diff
@@ -485,6 +485,8 @@ var ListActions = module.exports = kind(
 		var listAction, i;
 
 		this.listActionComponents = [];
+		this.$.listActions.destroyClientControls();
```

Comment:
@aarontam - Thanks for your comment. I'll make a PR for this and it would be like below.
```javascript
if (this.generated) {
    this.$.listActions.destroyClientControls();
}
```
Do I need to create a jira ticket for this? Or do we just review and merge it?